src/mgr/Action.h

```diff
@@ -70,7 +71,7 @@ class Action: public RefCountable
     StoreEntry *createStoreEntry() const; ///< creates store entry from params
 
     ///< Content-Type: header value for this report
-    virtual const char *contentType() const {return "text/plain;charset=utf-8";}
+    virtual const char *contentType(const String &) const {return "text/plain;charset=utf-8";}
```

Comment:
Please undo this API change (and its effects). The undocumented (and currently unused) String parameter is not needed for the Action to compute the Content-Type value based on the command parameters (e.g., the Accept request header). The Mgr::Action::fillEntry() code in this PR demonstrates how the Accept request header info can be extracted when needed.

Reply:
Hmm, it was needed for some reason I no longer recall. It is unused simply because the demo report conversion is gone.

Reply:
Done as of [8c9671e](https://github.com/squid-cache/squid/pull/70/commits/8c9671e8bf5843a22e2042a1e027feb67a9c5d96)